packages/ndla-ui/src/Gloss/Gloss.tsx

```diff
@@ -124,7 +124,9 @@ const Gloss = ({ title, glossData, audio }: Props) => {
                   </span>
                 )}
                 {glossData.wordClass && (
-                  <TypeSpan aria-label={t('gloss.wordClass')}>{t(`wordClass.${glossData.wordClass}`)}</TypeSpan>
+                  <TypeSpan aria-label={t('gloss.wordClass')}>
+                    {t(`wordClass.${glossData.wordClass.charAt(0).toUpperCase() + glossData.wordClass.slice(1)}`)}
```

Comment:
Dessuten har du satt uppercase her og ikkje lowercase.

Reply:
Swing and a miss